packages/@secretlint/formatter/src/formatters/table.ts

```diff
@@ -0,0 +1,154 @@
+"use strict";
+import { FormatterOptions } from "../types";
+import { SecretLintCoreResult, SecretLintCoreResultMessage } from "@secretlint/types";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+import chalk from "chalk";
+import { table } from "table";
+import stripAnsi from "strip-ansi";
+const pluralize = require("pluralize");
```

Comment:
We prefer to use `import` for consistency.
```suggestion
// @ts-expect-error: no types
import pluralize from "pluralize";
```